tests/unittests/test_cursor_pagination.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+from unittest import mock
+from tap_recharge.client import RechargeClient
+from tap_recharge.streams import Addresses
+
+COUNTER = 0
+
+def get(*args, **kwargs):
+    """
+        Function to return API response with 'next_cursor' for 2 responses and last response without 'next_cursor'
+    """
+    global COUNTER
```

Comment:
Add a reason for using a global variable in the comment.

Reply:
Updated the code to not use a global variable.